### DISCUSSION TITLE
Make Post Date, Expiry Date optional in publish form

### DISCUSSION THREAD

**Discussion by timkelty** [craftcms/cms#8887] [Ideas]
A bit of a companion to https://github.com/craftcms/cms/issues/6526…

As these terms don't always make sense for every content model, it would be nice to be able to optionally hide them from the publish screen.

**Comment by brandonkelly** [comment]
Yeah we’ll have to do something along these lines when we merge entries, categories, and tags.

**Reply by carlcs** [reply]
Would be awesome to see this included in Craft 4.4!
